data/json/furniture_and_terrain/terrain-walls.json

```
@@ -986,7 +986,7 @@
     "symbol": "LINE_OXOX",
     "color": "light_cyan",
     "move_cost": 0,
-    "coverage": 90,
+    "coverage": 25,
```

```suggestion
    "coverage": 30,
```
Changing them both to thirty to test if that will cause tests to fire.